torch/quantization/_equalize.py

```diff
@@ -0,0 +1,89 @@
+import torch
+import torch.nn as nn
+
+def channel_range(tensor, axis =0):
```

Comment:
NIT: Please dont use tensor as a variable name, torch.Tensor is a class in pytorch and the naming can be confusing